tests/integration/icontracts/contracts/storage_versions/storage_v0_2_1.py

```
@@ -0,0 +1,19 @@
+# v0.2.1
+# { "Depends": "py-genlayer:1jb45aa8ynh2a9c9xn3b7qqh8sm5q93hwfp7jqmwsfhh8jpz09h6" }
+
+from genlayer import *
```

Ruff (0.14.14)

[error] `from genlayer import *` used; unable to detect undefined names

(F403)